app/assets/javascripts/jquery_overrides.js

```diff
@@ -37,10 +37,20 @@ $.ajaxSetup({
     }),
     "text script": logError(function (text) {
```

Comment:
`logError` is a high order function that calls a function catching and logging any errors.

... and it's not the subject of this PR ;-)